### ISSUE TITLE
Make section headings linkable in Census

### ISSUE THREAD

**Issue by mheadd** [CityOfPhiladelphia/slash-data#24]
It would be useful if the section headers were linkable, like so:

http://cityofphiladelphia.github.io/slash-data/census/#pipeline

Any chance this could be added?

**Comment by timwis** [comment]
Good idea. Hm...looks like it conflicts with the javascript app's routing, which uses the hash. If you've already got the page loaded, appending #view_40 to the URL gets you to the pipeline, but it doesn't work as a fresh link: http://cityofphiladelphia.github.io/slash-data/census/#view_40 Can't think of a way around this :-/

**Comment by gsf** [comment]
Another reason to move away from the built-in Knack interface and just use the API.
